docs/dev-guide/tscript/debugging.mdx

```diff
@@ -190,13 +190,24 @@ The two main sources of dev-prod discrepancies are in bundling and connecting.
 
 #### Production bundling
 
-If you are getting errors like this:
+You may experience your Client sending stripped names as the Workflow "Type" when scheduling a Workflow.
+Webpack can change the Workflow's function name to something shorter.
+Temporal won't know how to handle the mismatch between the shorter name and the expect Workflow type.
+
+You may experience errors like this:
 
 ```
 Error: 3 INVALID_ARGUMENT: WorkflowType is not set on request.
 ```
 
-This is due to your bundler stripping out Workflow function names, which we rely on to set the "Workflow Type" in Temporal. Turn it off and it should work.
+Or you may see shorter names in the Temporal Service's Web UI when Webpack changed the Workflow's function name to something shorter, in this case the single letter 's':
+
+![Temporal Web UI showing stripped "Workflow Type" entries, in this case the single letter 's'](/img/stripped_workflow_types_in_webui.png)
+
+This issue can happen when your bundler strips out Workflow function names.
+Temporal relies on those names to set the "Workflow Type" in the Service Web UI.
+To prevent the build process from removing Workflow function names,  modify the webpack configuration file ( `webpack.config.js`) to set the Boolean that retains the original names in the `TerserPlugin` configuration section.
```

Comment:
```suggestion
To prevent the build process from shortening Workflow function names,  modify the webpack configuration file ( `webpack.config.js`) to set the Boolean that retains the original names in the `TerserPlugin` configuration section.
```

Changed to be more accurate, as it does not "remove" them. Oops.